File: BE-FindingJob/src/main/java/com/example/befindingjob/dto/NotificationEntityDTO.java
package com.example.befindingjob.dto;

import lombok.*;

@Data
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class NotificationEntityDTO {
    private Long id;
    private String content;
    private boolean read;
    private String createdAt;
    private UserDTO receiver;
    private UserDTO sender;
    private ApplicationDTO application;
}

File: JobHub/app/src/main/java/com/example/jobhub/dto/NotificationEntityDTO.kt
package com.example.jobhub.dto

data class NotificationEntityDTO(
    val id: Long? = 0,
    val content: String? = null,
    var read: Boolean,
    val createdAt: String? = null,
    val receiver: UserDTO? = null,
    val sender: UserDTO? = null,
    val application: ApplicationDTO? = null
)
